cloudevents/tests/data.py

```diff
@@ -21,6 +21,30 @@
 eventTime = "2018-10-23T12:28:23.3464579Z"
 body = '{"name":"john"}'
 
+test_cloudevent_headers = [
```

Comment:
We should test valid CloudEvents.

These are invalid CloudEvents because they don't have all 4 required headers.

Please add all required headers to the test cases.